scripts/release.ts

```diff
@@ -69,6 +82,20 @@ const createDraftPR = async (): Promise<void> => {
   exit(chalk.green(`Release ${nextVersion} created.`))
 }
 
+const createDraftHotfixPR = async (): Promise<void> => {
+  const currentBranch = await git().revparse(['--abbrev-ref', 'HEAD'])
+  const { messages } = await getCommits(currentBranch as GetCommitMessagesArgs)
+  // TODO(0xdef1cafe): parse version bump from commit messages
+  const formattedMessages = messages.map(m => m.replace(/"/g, '\\"'))
+  const nextVersion = await getNextReleaseVersion('minor')
+  const title = `chore: hotfix release ${nextVersion}`
```

Comment:
Note `hotfix release` and yes, we do maintain semantic versioning despite it being a hotfix